### ISSUE TITLE
Unraid

### ISSUE THREAD

**Issue by bbakermmc** [JetBrains/rider-plugins-docker-image#1]
Looking over the docker files, does it install rider?  I didnt see it in the packages.

Also could you make a version that could be run in Unraid?
EX: https://github.com/binhex/docker-templates/blob/master/binhex/intellij.xml
https://github.com/binhex/arch-intellij/blob/master/build/root/install.sh

**Comment by IvanPashchenko** [comment]
Nope, these images do not install Rider. They are helping us to have a stable build environment for Rider's bundled plugins (e.g. Unity, F#) and not meant to have any IDEs — only build tools and SDKs that are required for build are present.

**Comment by bbakermmc** [comment]
@IvanPashchenko Ahhhh, I was looking for a pre configured environment to develop from :), spin up a fresh docker for each dev on a cluster and then connect to it using like vnc, then the IDE can be hosted on a beefier machine, plus great if you want to be remote and connect from multiple machines.  @binhex has the intellij for unraid, but not rider, and hasnt responded to me for doing a rider one :(

**Comment by binhex** [comment]
@bbakermmc i MIGHT have a xmas present for you :-), no guarantees depends on free time, but im looking at rider.

**Comment by bbakermmc** [comment]
Nice!  I’ll take New Years also lol.  The JB ides are really nice.  I only use VS when I have too for some Azure/SSDT stuff.

Sent from my iPhone

On Dec 23, 2018, at 7:09 AM, binhex <notifications@github.com<mailto:notifications@github.com>> wrote:


@bbakermmc<https://github.com/bbakermmc> i MIGHT have a xmas present for you :-), no guarantees depends on free time, but im looking at rider.
